api/source/specification/stig-manager.yaml

```diff
@@ -1305,6 +1305,350 @@ paths:
+    get:
+      tags:
+        - Collection
+      summary: Return the keys of the provided Collection metadata
+      operationId: getCollectionMetadataKeys
+      responses:
+        '200':
+          description: MetadataKeys response
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/MetadataKey'
+        default:
+          description: unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+      security:
+        - oauth:
+            - 'stig-manager:collection:read'
+  '/collections/{collectionId}/metadata/keys/{key}':
+    parameters:
+      - $ref: '#/components/parameters/ElevateQuery'
+      - $ref: '#/components/parameters/CollectionIdPath'
+      - $ref: '#/components/parameters/MetadataKeyPath'
+    get:
+      tags:
+        - Collection
+      summary: Return the value of the provided Collection metadata key
+      operationId: getCollectionMetadataValue
+      responses:
+        '200':
+          description: MetadataValue response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/MetadataValue'
+        default:
+          description: unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+      security:
+        - oauth:
+            - 'stig-manager:collection:read'
+    put:
+      tags:
+        - Collection
+      summary: Set the value of the provided Collection metadata key to the provided string
+      operationId: putCollectionMetadataValue
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/MetadataValue'
+      responses:
+        '204':
+          description: Empty to avoid large response after putting key with large value
+        default:
+          description: unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+      security:
+        - oauth:
+            - 'stig-manager:collection'
+    delete:
+      tags:
+        - Collection
+      summary: Remove the provided Collection metadata key
+      operationId: deleteCollectionMetadataKey
+      responses:
+        '204':
+          description: Empty to avoid large response after deleteing key with large value
+        default:
+          description: unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+      security:
+        - oauth:
+            - 'stig-manager:collection'
+  '/assets/{assetId}/metadata':
+    parameters:
+      - $ref: '#/components/parameters/ElevateQuery'
```

Comment:
Please remove the `elevate` param from this endpoint

Reply:
removed